### ISSUE TITLE
Update colours

### ISSUE THREAD

**Issue by Turbotailz** [syscraft-mc/syscraft.dev#2]
The site is using default colour scheme provided by the Astro template.

Need to go through and update the theme with Syscraft brand colours.

**Comment by NotGeri** [comment]
I couldn't find anything on colours but just using the one from the Discord banner doesn't look _too_ horrible:

![img](https://i.geri.dev/jqCP7wlLdXrH.png)

Although even the current theme is pretty readable and solid IMO.

**Comment by Turbotailz** [comment]
Yeah I was meant to update this issue with the proper brand colours 😅
